Packs/UrlScan/Integrations/UrlScan/UrlScan.yml

```diff
@@ -337,6 +337,21 @@ script:
     - contextPath: URLScan.EffectiveURL
       description: Effective URL of the original URL.
       type: string
+    - contextPath: URL.ASN
+      description: The URL ASN
+      type: String
+    - contextPath: URL.FeedRelatedIndicators.value
+      description: Indicators that are associated with the URL
+      type: String
+    - contextPath: URL.FeedRelatedIndicators.type
+      description: the type of the indicators that are associated with the URL
+      type: String
+    - contextPath: URL.Geo.Country
+      description: The URL country
+      type: String
+    - contextPath: URL.RelatedASNs
```

Comment:
Is this returned like this from the server? I don't see the RelatedASNs here on code

Reply:
It was a mistake, I fixed